views/includes/page-templates/beneficial-owner-nature-of-control-types.html

```diff
@@ -9,9 +9,16 @@
 {% set errorMessage = errors.beneficial_owner_nature_of_control_types if errors %}
 {% include "includes/inputs/nature-of-control-checkboxes/beneficial-owner-nature-of-control-types.html" %}
 
-{% if not isBeneficialOwnerGov and not entity_number %}
-  {% set errorMessage = {} if errors.beneficial_owner_nature_of_control_types %}
-  {% include "includes/inputs/nature-of-control-checkboxes/trustees-nature-of-control-types.html" %}
+{% if entity_number %}
+  {% if not isBeneficialOwnerGov and pageParams.isTrustFeatureEnabled %}
```

Comment:
You have access to the flag through nunjucks global variables, defined we have: `nunjucksEnv.addGlobal("OE_CONFIGS", config);` so in the view you can just do `OE_CONFIGS.<flagname>` and this avoids changing the controllers